lib/agent/providers/geo/strategies.js

```diff
@@ -0,0 +1,121 @@
+"use strict"
+
+var join = require('path').join,
+  needle = require('needle'),
+  get_agent = require('random-ua').generate,
+  platform = require(join(__dirname, process.platform)),
+  providers = require('./../../providers'),
+  logger = require('../../common').logger.prefix('geo');
+
+function geoip(cb) {
+
+  logger.debug("Getting location via geoip");
+
+  needle.get('http://ipinfo.io/geo', function(err, resp, body) {
+    if (!body || !body.loc) {
+      return cb(err || new Error('Unable to get location from IP.'));
+    }
+
+    logger.debug("Got location via geoip");
+
+    var res = {
+      lat: parseFloat(body.loc.split(',')[0]),
+      lng: parseFloat(body.loc.split(',')[1]),
+      method: 'geoip'
+    }
+
+    cb(null, res);
+  })
+}
+
+function google(cb) {
+
```

Comment:
Extra lines, not consistent with previous style. Review extra lines moving down as well.